snoop/data/filesystem.py

```diff
@@ -112,8 +114,8 @@ def create_directory_children(directory, children):
         child_files = []
         for item in children:
             if item['type'] == 'file':
-                blob = models.Blob.objects.get(pk=item['blob_pk'])
-                file = create_file(directory, item['name'], blob)
+                original = models.Blob.objects.get(pk=item['blob_pk'])
```

Comment:
this function should still work on `File.blob`, in case we want to mangle archives somehow